### DISCUSSION TITLE
Highlight without copying text

### DISCUSSION THREAD

**Discussion by jcesguerram** [RyotaUshio/obsidian-pdf-plus#64] [Q&A]
Great to see the plugin made it to the plugin store! (or whatever it's called)
i've already tried it out but from what i can see it's still not posiible to highlight without copying the texte somewhere. is that right?

**Comment by RyotaUshio** [comment]
In fact, you can! PDF++ allows you to add highlights directly to PDF files. Highlights created as such are embedded inside PDFs themselves and therefore are visible without pasting links to them (and even outside Obsidian).

To be able to do this, you need to enable PDF editing in the following setting and fill in the annotation author form:

<img width="790" alt="image" src="https://github.com/RyotaUshio/obsidian-pdf-plus/assets/72342591/3ca4c656-3e18-4a48-917c-e1f20788e05a">

After that, you can do this:

https://github.com/RyotaUshio/obsidian-pdf-plus/assets/72342591/64fc8e10-0b9f-4ba8-9faa-a79abca15f99

I used the color palette in this video but you can also use the hotkeys.
